packages/web/src/pages/asset-detail-page/components/BalanceSection.tsx

```diff
@@ -226,28 +226,32 @@ const BalanceSectionContent = ({ mint }: AssetDetailProps) => {
   const logoURI = coin.logoUri
 
   return (
-    <Paper ph='xl' pv='l'>
-      <Flex direction='column' gap='l' w='100%'>
-        {!tokenBalance?.balance ||
-        Number(tokenBalance.balance.toString()) === 0 ? (
-          <ZeroBalanceState
-            ticker={ticker}
-            logoURI={logoURI}
-            onBuy={handleAddCash}
-            onReceive={handleReceive}
-          />
-        ) : (
-          <HasBalanceState
-            ticker={ticker}
-            logoURI={logoURI}
-            onBuy={handleBuySell}
-            onSend={handleSend}
-            onReceive={handleReceive}
-            mint={mint}
-          />
-        )}
-      </Flex>
-    </Paper>
+    <>
+      <Paper ph='xl' pv='l'>
+        <Flex direction='column' gap='l' w='100%'>
+          {!tokenBalance?.balance ||
+          Number(tokenBalance.balance.toString()) === 0 ? (
+            <ZeroBalanceState
+              ticker={ticker}
+              logoURI={logoURI}
+              onBuy={handleAddCash}
+              onReceive={handleReceive}
+            />
+          ) : (
+            <HasBalanceState
+              ticker={ticker}
+              logoURI={logoURI}
+              onBuy={handleBuySell}
+              onSend={handleSend}
+              onReceive={handleReceive}
+              mint={mint}
+            />
+          )}
+        </Flex>
+      </Paper>
+
+      <SendTokensModal />
```

Comment:
why is this necessary.. is this a new pattern

Reply:
no it's not great catch